internal/test/wmcb/wmcb_test.go

```diff
@@ -92,6 +105,9 @@ func TestWMCB(t *testing.T) {
 		payloadDirectory: remoteDir,
 		cniDirectory:     winCNIDir,
 	}
+	if !*dockerRuntime {
+		srcDestPairs[containerdDir] = winContainerdDir
+	}
```

Comment:
You're dereferencing a pointer without checking if it is nil

Reply:
Its better to use https://pkg.go.dev/flag#BoolVar
so you dont have to check if it is nil or not